### ISSUE TITLE
Max pos size and equity

### ISSUE THREAD

**Issue by rdvnszn1** [nkaz001/algotrading-example#1]
Hi I like your works but can not understand the return with  equity  and max_position,   how do you calculate the leverage and balance you need ?

**Comment by nkaz001** [comment]
I didn't take the leverage into account in the code. In live trading, if you want to use 1x leverage, you have to maintain your balance close to max_position. If you want to use 2x leverage, it means you are allowed to maintain your balance to only half of max_position. In short, your leverage is decided by the ratio of max_position over your actual balance. And of course, you have to be careful of the change of XBTUSD value as your actual balance is in BTC and your position is in USD in BitMEX. It is much cleaner in other futures that the profit is settled in USD such as Binance USD(S)-M futures.

**Comment by rdvnszn1** [comment]
but then how you calculate return in your  backtest ?

**Comment by nkaz001** [comment]
Now I got it. There was something I understood wrong and I didn't carefully calculate. I calculate my return just by my equity changes.

 I set the start equity 1 BTC and use max_position 10000 USD. XBTUSD value is around 3000~4000 on the start date of the backtest. So the stating leverage is around 2.5 ~ 3.3 and +1 for hedging.

But I think I need to change the return calculation.
Thanks for pointing it out. Any suggestion would be appreciated.
